src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java

```diff
@@ -26,6 +26,13 @@ public void doNotAcceptTermsAndConditions() {
         termsAndConditionsPage.clickContinue();
     }
 
+    @When("^I double the click send application button in terms and conditions page")
```

Comment:
Should be I double click the ...

Reply:
Yes, it should be `I double click the send application button on the terms and conditions page`.

Reply:
Sorry i did not understand the comment @Adeel @Mike it was "I double click the .." later changed to "I double the click .." as per above comment

Reply:
I got it thanks @mikeSmith08 . applied changes